### PR TITLE
json spec - add description for autoscaling

### PR DESCRIPTION
**Pull request by jakelandis**
Allow the autoscaling REST specs to pass the schema validation.

### CODE REVIEW

File: x-pack/plugin/src/test/resources/rest-api-spec/api/autoscaling.put_autoscaling_policy.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "autoscaling.put_autoscaling_policy":{
     "documentation":{
-      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/autoscaling-put-autoscaling-policy.html"
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/autoscaling-put-autoscaling-policy.html",
+      "description": "Create a new autoscaling policy."
```
Comment (henningandersen) [inline x-pack/plugin/src/test/resources/rest-api-spec/api/autoscaling.put_autoscaling_policy.json:5]: nit for consistency:

```suggestion
      "description": "Creates a new autoscaling policy."
```

**Review by henningandersen — APPROVED** [review]
LGTM

**Comment by jakelandis** [comment]
@elasticmachine run elasticsearch-ci/bwc

**Comment by jakelandis** [comment]
@elasticmachine update branch

**Pull request merged**